src/features/AnnouncementModal/AnnouncementModal.tsx

```diff
@@ -58,6 +60,18 @@ export const AnnouncementModal = ({
 
   const { title, description, image, tags } = announcements[currActiveIdx]
 
+  let descComponent: JSX.Element = <></>
+  if (typeof description === "string") {
+    descComponent = (
+      <Text textStyle="body-1" color="base.content.default">
+        {" "}
+        {description}{" "}
```

Comment:
I think these spaces should be removed. It is causing the description to not be aligned correctly:

![Screenshot 2023-09-26 at 10 35 03 AM](https://github.com/isomerpages/isomercms-frontend/assets/27919917/9075e148-4457-488a-9434-63f791cabb61)

Reply:
wah good catch 
db12145f288fed970215bb9fef7225d5a6c39c34